var/spack/repos/builtin/packages/rocm-clang-ocl/package.py

```diff
@@ -80,12 +80,14 @@ def cache_test_sources(self):
         install test subdirectory for use during `spack test run`."""
         self.cache_extra_test_sources([self.test_src_dir])
 
-    def test(self):
+    def test_make(self):
+        """Test make and cmake"""
         test_dir = join_path(self.test_suite.current_test_cache_dir, self.test_src_dir)
         with working_dir(test_dir, create=True):
```

Comment:
The working directory is automatically created so no reason to pass `create=True`.